### PR TITLE
Return correct OS version on iOS

### PR DESCRIPTION
**Pull request by ann0see**
**Short description of changes**

Returns correct OS version for iOS servers.

CHANGELOG: iOS: Return correct operating system version if run as server

**Context: Fixes an issue?**

Fixes: #3359 

**Does this change need documentation? What needs to be documented and how?**

No

**Status of this Pull Request**

Review

**What is missing until this pull request can be merged?**

Tested on macOS and iOS, but I want to test the GitHub produced macOS build too. 

## Checklist

-  [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/main/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
-  [x] I tested my code and it does what I want
-  [x] My code follows the [style guide](https://github.com/jamulussoftware/jamulus/blob/main/CONTRIBUTING.md#source-code-consistency) 
-  [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
-  [x] I've filled all the content above

### CODE REVIEW

**Comment by pljones** [comment]
> Tested on macOS and iOS, but I want to test the GitHub produced macOS build too.

Is checking the GitHub build done now?

File: src/util.h [hunk 1]
```diff
@@ -1098,8 +1098,10 @@ class COSUtil
     {
 #ifdef _WIN32
```
Comment (pljones) [inline src/util.h:1099]: Are the Windows and Android ones okay?  Is the `Q_` a "Qt" thing that we can use for all platforms?  What's the `OT_` thing?

**Comment by ann0see** [comment]
> Is checking the GitHub build done now?

I‘ll need to re check.

File: src/util.h (hunk 1 above)
Comment (ann0see) [reply src/util.h:1099]: Check the enum above

Comment (ann0see) [reply src/util.h:1099]: Windows and android are ok, but probably worth changing to the Qt style.

**Comment by ann0see** [comment]
Ok. The GitHub build works as expected.

**Review by softins — APPROVED** [review]
The MacOS Legacy artifact shows MacOS correctly. I can't test the iOS build, but the code looks sensible, and I'm happy to approve.

**Review by pljones — APPROVED** [review]

File: src/util.h (hunk 1 above)
Comment (ann0see) [reply src/util.h:1099]: It seems as if we don't use Q_OS_WIN anywhere. Android is now in the same style as used in other places.

**Comment by ann0see** [comment]
@pljones @softins last minute android changes. Please recheck.

**Review by pljones — APPROVED** [review]
Not sure I like the "or" approach - but it should be okay.  (And Android is barely supported...)

**Comment by ann0see** [comment]
It's what I see throughout the code...

**Pull request merged**